docs/blog/20230525-Daeraxa-Survey2-Results.md

```diff
@@ -0,0 +1,103 @@
+---
+title: "Survey Results: Project feedback"
+author: Daeraxa
+date: 2023-05-25
+category:
+  - survey
+tag:
+  - community
+  - feedback
+---
+
+Here we have the results and comments of our second community survey!
+
+<!-- more -->
+
+Thanks to everyone who took part in this survey, we got some great responses, again showing how effective this style of community engagement can be.
+
+The reason for this survey was to gauge some early feedback on a bunch of changes we are looking at making in various areas of the project. We really need to make sure that it is worth making these changes, the last thing we want is to make changes that nobody actually wants that end up making things worse.
+
+Honestly there isn't much more to say as there wasn't really an overarching theme for this one like there was with our [first survey](https://pulsar-edit.dev/blog/20230326-Daeraxa-Survey1-Results.html) so let's just get on with it shall we?
+
+## Iconography
+
+Our first question was asking about upgrading our icon theme in Pulsar. We are updating our [Octicons](https://primer.style/design/foundations/icons/) version bundled with Pulsar to take advantage of changes made between v4.4.0 from 2016 to the current v18.3.0. As part of this we want to explore actually using these new icons in our default UI theme. The questions below are voting on preference between two styles of the new icons against the current theme:
```

Comment:
```suggestion
Our first question was asking about upgrading our icons in Pulsar. We are updating our [Octicons](https://primer.style/design/foundations/icons/) version bundled with Pulsar to take advantage of changes made between v4.4.0 from 2016 to the current v18.3.0. As part of this we want to explore actually using these new icons in our default UI theme. The questions below are voting on preference between two styles of the new icons against the current theme:
```

Feel like this is a little bit more accurate, as it's technically the font file, rather than a theme. Since theme would assume it's like any Pulsar theme that can be totally changed out, or reverted easily, which isn't really the case.